solutions/gold/usaco-1043.mdx

```diff
@@ -0,0 +1,203 @@
+---
+id: usaco-1043
+source: USACO Gold 2020 US Open
+title: Exercise
+author: Ryan Chou, Kevin Sheng
+---
+
+<Spoiler title="Hint 1">
+
+How do we know whether a "$k$-step" path is achievable?
+
+</Spoiler>
+
+<Spoiler title="Hint 1 (answer)">
+
+After examining the sample input (more specifically the $6$-step case in the problem statement),
+we can see that the cows move in disjoint cycles.
+
+![Cycles of the sample input.](/usaco-1043/hint1.png)
+
+Since in each cycle $i$, the cows take $\texttt{len}(i)$ steps to return to their original
```

Comment:
The "since in each cycle" part seems a little incomplete... it doesn't really make sense here

Reply:
wdym by that

Reply:
jesse u gotta respond to reviews 😭

Reply:
okay broski im js gna merge rn -- if you wanna make any changes do another pr